pkg/encryption/encryption.go

```diff
@@ -46,6 +46,8 @@ func Encrypt(data []byte, cipher storj.CipherSuite, key *storj.Key, nonce *storj
 		return EncryptAESGCM(data, key, ToAESGCMNonce(nonce))
 	case storj.EncSecretBox:
 		return EncryptSecretBox(data, key, nonce)
+	case storj.EncURLSafeBase64:
```

Comment:
why doesn't this work?

Reply:
Afaik this is for encrypting data as opposed to paths. We don't want anyone accidentally url-base64-encoding data they intend to be encrypted, instead of actually encrypting it.